app/views/training/questions/show.html.slim

```diff
@@ -16,6 +16,7 @@
       - if content.formative_question? && current_user_response.responded?
         #formative-results= render 'assessment_banner'
 
+      = render 'training/pages/timestamp' if content.formative_question?
```

Comment:
I'd recommend you put this on two lines even though it is really short. Using nesting and conditions close to the left is a good pattern to follow